src/vcSettings.cpp

```diff
@@ -567,7 +564,13 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
       udStrcpy(pSettings->loginInfo.email, data.Get("login.username").AsString());
 
     // Camera
-    pSettings->camera.moveSpeed = data.Get("camera.moveSpeed").AsFloat(10.f);
+    for (int v = 0; v < vcMaxViewportCount; v++)
+    {
+      pSettings->camera.moveSpeed[v] = data.Get("camera.moveSpeed[%d]", v).AsFloat(10.f);
```

Comment:
These all need to be backwards compatibility, use the old saved settings as input:

E.g.:
```
pSettings->camera.moveSpeed[v] = data.Get("camera.moveSpeed[%d]", v).AsFloat(data.Get("camera.lensId"));
```
and
```
pSettings->camera.mapMode[v] = data.Get("camera.mapMode[%d]", v).AsBool(data.Get("viewports.viewport[%d].mapMode", v).AsBool(false));
```

Reply:
Fixed